### DISCUSSION TITLE
Przetwarzanie obrazu

### DISCUSSION THREAD

**Discussion by ghost** [AzureCommunityPL/group#44] [Pomoc] [answered]

**Comment by szympulka** [comment]
@culespl Zmienne środowiskowe możesz wrzucić ręcznie i efekt będzie ten sam. Chociaż możesz też w kodzie bezpośrednio stringa użyć do tego. [link](https://support.microsoft.com/pl-pl/help/310519/how-to-manage-environment-variables-in-windows-xp)

Jeżeli chodzi o samo przetwarzanie obrazów to tutaj masz [sample](https://docs.microsoft.com/pl-pl/azure/cognitive-services/computer-vision/tutorials/csharptutorial) oraz [moduł learning](https://docs.microsoft.com/pl-pl/learn/browse/?terms=Computer%20Vision) i ogólnie [dokumentacja](https://docs.microsoft.com/pl-pl/azure/cognitive-services/computer-vision/). Używałem tych linków kiedyś i pierwsze kroki powinieneś w nich postawić.

**Comment by MarczakIO** [comment, marked as answer]
Ciezko pomoc z kazdym pytanie bo strasznie chaotycznie opisales problemy. Tego ze zmiennymi srodowiskowymi nawet nie zrozumialem.

> Dane sa niektore juz chyba przedawnione, gdyz uzywane biblioteki sa nie dostepne.

Artykul aktualizowany 27.08 wiec raczej aktualny.

> Kolejna sprawa jak dodac biblioteke, ktora jest niedostepna z poziomu Manage NuGet Packages

Jesli chodzi o paczki to

1. Jako ze paczka jest w preview co widac na stronie, trzeba wybrac 'include prerelease' checkboxa w managerze
![image](https://user-images.githubusercontent.com/32310272/91839074-bd8d9c00-ec4e-11ea-8523-4daf56d0bfb1.png)
2. Link ktory wkleiles daje Ci te komendy do odpalenia wiec nie musisz package managera uzywac

> Kolejny problem z przetwarzaniem obrazu, robie wszystko jak w instrukcji:

Robilem kiedys ten quickstart i bylo ok. Moze jednak cos opusciles.

**Reply by wi5nia** [reply]
Tez potwierdzam, ze dziala wiec na spokojnie od poczatku bo wszystko dziala.

**Reply by ghost** [reply]
Kurcze przetwarzanie robie wg tego linku wszystko 1:1
https://github.com/microsoft/AcademicContent/blob/master/Labs/Azure%20Services/Azure%20Storage/Azure%20Storage%20and%20Cognitive%20Services%20(MVC).md#Exercise3
i wszystko działa poza megadanymi no i jeszcze nie robilem wyszukiwania.
Nawet po pobraniu projektu z github 
https://github.com/microsoft/AcademicContent/tree/master/Labs/Azure%20Services/Azure%20Storage/Solutions/Intellipix
oczywiscie po zmianach kluczy, dodaniu wszystkiego do Portalu Azure, itp jest ten sam problem działa ale bez megadanych.
Robilem od poczatku do konca juz kilkanascie razy.

**Reply by MichalJankowskii** [reply]
Ojej, ale gruby jest tam przykład.

Można to spróbować na czymś prostszym - [Quickstart: Use the Computer Vision client library](https://docs.microsoft.com/en-us/azure/cognitive-services/computer-vision/quickstarts-sdk/client-library?pivots=programming-language-csharp)

**Reply by MichalJankowskii** [reply]
I trochę należy uważać z bibliotekami -preview.

**Reply by ghost** [reply]
i mi rowniez juz dziala! dzieki za pomoc. Lece z kolejnymi mam nadzieje ze obedzie sie bez pytan
